openapi.yml

```diff
@@ -3055,6 +3055,138 @@ paths:
                     type: string
                 in: path
                 required: true
+    '/locations/{locationId}/deliveries':
+        get:
+            tags:
+                - Deliveries
+                - Locations
+            responses:
+                '200':
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/PaginatedDeliveries'
+                            examples:
+                                default:
+                                    value:
+                                        deliveries:
+                                            -
+                                                location_id: 95
+                                                host_id: 73
```

Comment:
Can I get the host object returned instead of an ID? Same with location.

Reply:
Yes, but I'll need to change some things soon to use an `include` param to get related objects